src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -321,8 +321,11 @@ func PachctlCmd() *cobra.Command {
 Environment variables:
   PACH_CONFIG=<path>, the path where pachctl will attempt to load your pach config.
   JAEGER_ENDPOINT=<host>:<port>, the Jaeger server to connect to, if PACH_TRACE is set
-  PACH_TRACE={true,false}, If true, and JAEGER_ENDPOINT is set, attach a
-    Jaeger trace to any outgoing RPCs
+  PACH_TRACE={true,false}, if true, and JAEGER_ENDPOINT is
+    set, attach a Jaeger trace to any outgoing RPCs.
```

Comment:
Somewhat short line-wrapping going on here.  We should pick a column limit and try to fill it out.

Reply:
Yeah, I think must've been trying to lay out the lines manually and made a mistake. There was certainly no deliberate effort to keep this line super short. I reflowed these lines and they should be fixed (all ~80 characters)